luminance-gl/src/gl33/pipeline.rs

```diff
@@ -124,7 +125,16 @@ where
       } else {
         0
       };
+      state.set_scissor_state(ScissorState::Off);
```

Comment:
So I have a question. If we have enabled the scissor test… should we not keep it enabled when clearing, so that we don’t have to clean something we know we haven’t written to?

Reply:
I think leaving the scissor enabled but setting the scissor region to the framebuffer size before clearing would work. One less call to GL as well.

Reply:
We could eventually provide an API for the user to provide their own scissor region when clearing.

Reply:
But isn’t that what they already expect? I don’t really use that feature so I don’t know the different possible needs. It would be cool to list them somewhere — the original issue is fine.